### ISSUE TITLE
Deployment to Github Pages fails 

### ISSUE THREAD

**Issue by kfrie** [squidfunk/mkdocs-material#683]
## Description

Deployment to Github Pages fails
 
### Expected behavior

Built documentation on gh-pages branch

### Actual behavior

```
INFO    -  Cleaning site directory
INFO    -  Building documentation to directory: /docs/site
INFO    -  Copying '/docs/site' to 'gh-pages' branch and pushing to GitHub.
git: 'fast-import' is not a git command. See 'git --help'.
```
### Steps to reproduce the bug

```
docker run --rm -it -v ${userprofile}/.ssh:/root/.ssh -v ${PWD}:/docs squidfunk/mkdocs-material gh-deploy
```

### Package versions

Docker image `squidfunk/mkdocs-material:2.5.3`

### Project configuration

``` yaml
site_name: SAFE Documentation

theme:
  name: 'material'

# Repository
repo_name: SAFE-Documentation
repo_url: https://github.com/kfrie/SAFE-documentation/
edit_uri: edit/master/docs/

markdown_extensions:
  - codehilite

pages:
- Home: 'index.md'
- Use with CSS Frameworks:
    - 'Wrap React Components': 'wrap-react-components.md'
```

### System information
* Windows 10 Education Version 10.0.16299 Build 16299

**Comment by squidfunk** [comment]
Thanks for filling the issue. Alpine Linux moved `git-fast-import` into a separate package to reduce size. I will add it to the `Dockerfile` and make a bugfix release.

**Comment by kfrie** [comment]
@squidfunk Thank you very much for the super fast support!

**Comment by squidfunk** [comment]
Fixed in #685, will be released as 2.5.4 shortly.
